.github/workflows/gpuci.yml

```diff
@@ -0,0 +1,41 @@
+name: "GPU CI/CD"
+
+on:
+  pull_request:
+    branches:
+      - 'main'
+      - 'r**'
+    types: [ labeled, synchronize ]
+
+jobs:
+  gpu-test:
+    runs-on: self-hosted
+    if: ${{ github.event.label.name == 'gpuci' || github.event_name == 'synchronize' }}
+
+    steps:
+    - name: Run nvidia-smi test
+      run: |
+        whoami
+        nvidia-smi
+
+    - name: Checkout the PR code
+      uses: actions/checkout@v3
+
+    - name: Set up Python 3.10
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
+    - name: Install NeMo-Curator and PyTest
+      run: |
+        pip install wheel cython
```

Comment:
Might be a silly question, but do we need `wheel`? I saw it in [test.yml](https://github.com/NVIDIA/NeMo-Curator/blob/main/.github/workflows/test.yml#L42) which is why I added it.

Reply:
@praateekmahajan might have run into this sooner, but iirc wheel is needed for pycld2 or fastext(don't recall which on) to install as expected.

Reply:
I don't think we need `wheel`, or atleast I haven't run into it yet when building a container

Let's try removing it and then see?

Reply:
Cool, looks like it's able to work without it.